samples/client/petstore/kotlin-retrofit2-kotlinx_serialization/src/main/kotlin/org/openapitools/client/infrastructure/ApiClient.kt

```diff
@@ -72,11 +72,16 @@ class ApiClient(
         authNames: Array<String>
     ) : this(baseUrl, okHttpClientBuilder) {
         authNames.forEach { authName ->
-            val auth = when (authName) {
-                "petstore_auth" -> OAuth(OAuthFlow.implicit, "http://petstore.swagger.io/api/oauth/dialog", "", "write:pets, read:pets")"api_key" -> ApiKeyAuth("header", "api_key")
```

Comment:
I think that this was also incorrect kotlin code due to the lack of line break before "api_key"

Reply:
agreed. the CI should have caught the issue. let me take a look tomorrow or later this week

Reply:
the code still compiles in the latest master so the CI is doing the right thing

Reply:
i cannot come up with a way off the top of my head to catch this issue easily moving forward. will merge this for the time being and later add a test (hopefully)